src/browser/pages/vscode.ts

```diff
@@ -16,7 +63,7 @@ try {
         return cb(undefined, result)
       }
       // FIXME: Only works if path separators are /.
-      const path = nlsConfig._resolvedLanguagePackCoreLocation + "/" + bundle.replace(/\//g, "!") + ".nls.json"
+      const path = nlsConfig?._resolvedLanguagePackCoreLocation + "/" + bundle.replace(/\//g, "!") + ".nls.json"
```

Comment:
Another `?` we can remove. Otherwise looks great! Feel free to merge.

Reply:
Nice! I'll remove this an merge. Thanks for the stellar feedback man!